tests/integration/coordinator/plugins/clickhouse/conftest.py

```diff
@@ -564,6 +564,7 @@ def create_astacus_configs(
             ),
             node=NodeConfig(
                 root=clickhouse_service.data_dir,
+                db_path=storage_path,
```

Comment:
This breaks the tests, because then all nodes share the same folder for their db, and when they snapshot at the same time they open and try to overwrite the same files (which sqlite blocks, but it still fails).

Doing something like that fixes the tests:
```python
    node_db_paths = [storage_path / f"db_{node_port}" for node_port in node_ports]
    for node_db_path in node_db_paths:
        node_db_path.mkdir(exist_ok=True)
```